inference-engine/src/vpu/graph_transformer/src/stages/loop_start.cpp

```diff
@@ -42,9 +42,17 @@ class LoopStart : public StageNode {
     }
 
     void serializeParamsImpl(BlobSerializer& serializer) const override {
-        serializer.append(attrs().get<uint32_t>("iterations-count"));
+        int32_t iterations_count = attrs().has("batchId") ? -1 : attrs().get<uint32_t>("iterations-count");
```

Comment:
Do we have possibilities to define `-1` as a special constant?

Reply:
Added special constant